raiden-dapp/src/services/raiden-service.ts

```diff
@@ -173,6 +179,11 @@ export default class RaidenService {
       let deniedReason: DeniedReason;
       if (e.message && e.message.indexOf('No deploy info provided') > -1) {
         deniedReason = DeniedReason.UNSUPPORTED_NETWORK;
+      } else if (
+        e.message &&
```

Comment:
You should be able to do `e.code === 'RDN_STATE_MIGRATION'`

Reply:
Nein, that did not work for me. :)